carma-messenger-core/traffic_incident/src/traffic_incident_node.cpp

```diff
@@ -59,6 +60,8 @@
               }
             return true;
         });
+    
+    ros::CARMANodeHandle::spin();
     ROS_INFO_STREAM("Traffic Incident node is initialized...");
```

Comment:
Let's move this log line above or it will never print.

Reply:
https://github.com/usdot-fhwa-stol/carma-messenger/pull/88/commits/07099bf68c01f8bd8655565517b452ad53e23a2b